packages/wasm-utils/src/note/mod.rs

```diff
@@ -197,32 +264,48 @@ impl JsNoteBuilder {
 		Self::default()
 	}
 
-	pub fn prefix(&mut self, prefix: Prefix) -> Result<(), JsValue> {
-		let prefix: String = JsValue::from(&prefix)
+	pub fn protocol(&mut self, protocol: Protocol) -> Result<(), JsValue> {
+		let protocol: String = JsValue::from(&protocol)
 			.as_string()
-			.ok_or(OpStatusCode::InvalidNotePrefix)?;
-		let note_prefix: NotePrefix = prefix.as_str().parse().map_err(|_| OpStatusCode::InvalidNotePrefix)?;
-		self.prefix = Some(note_prefix);
+			.ok_or(OpStatusCode::InvalidNoteProtocol)?;
+		let note_protocol: NoteProtocol = protocol
+			.as_str()
+			.parse()
+			.map_err(|_| OpStatusCode::InvalidNoteProtocol)?;
+		self.protocol = Some(note_protocol);
 		Ok(())
 	}
 
 	pub fn version(&mut self, version: Version) -> Result<(), JsValue> {
 		let version: String = JsValue::from(&version)
 			.as_string()
 			.ok_or(OpStatusCode::InvalidNoteVersion)?;
-		let note_version: NoteVersion = version.as_str().parse().map_err(|_| OpStatusCode::InvalidNotePrefix)?;
+		let note_version: NoteVersion = version
+			.as_str()
+			.parse()
+			.map_err(|_| OpStatusCode::InvalidNoteProtocol)?;
```

Comment:
Should `OpStatusCode` be `OpStatusCode::InvalidNoteVersion` not `OpStatusCode::InvalidNoteProtocol` ?